spec/reservation_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'spec_helper'
+
+
+#date structure enter: day, month, year for Date.Parse format
+
+describe 'Reservation Class' do
+  before do
+    @hotel = Hotel::Hotel.new
+    @start_date = Date.parse("5/9/2018")
```

Comment:
Instead of going through an instance of `Hotel` in these tests, you should be creating instances of the `Reservation` class directly with `Reservation.new`.